internal/pkg/daemon/enricher/container.go

```diff
@@ -77,7 +88,7 @@ func (e *Enricher) getNodeContainers(logger logr.Logger, nodeName string) (map[s
 						continue
 					}
 
-					containers[rawContainerID] = containerInfo{
+					containers[rawContainerID] = &containerInfo{
```

Comment:
why don't we update the cache already here? This will cache all the container infos that have been found already, wouldn't it?

Reply:
Yes I think this makes sense, changed as suggested :+1: